src/topp/MSGFPlusAdapter.cpp

```diff
@@ -502,6 +509,8 @@ class MSGFPlusAdapter :
     {
       process_params << "-mod" << mod_file.toQString();
     }
+    
+    legacy_conversion_ = (getStringOption_("legacy_conversion") == "true");
```

Comment:
bool legacy_conversion_ = getFlag_("legacy_conversion"); to avoid such conversions